src/main/java/de/craften/ui/swingmaterial/Utils.java

```diff
@@ -0,0 +1,158 @@
+package de.craften.ui.swingmaterial;
+
+import java.awt.Color;
+import java.awt.Dimension;
+import java.awt.Frame;
+import java.awt.GraphicsConfiguration;
+import java.awt.GraphicsDevice;
+import java.awt.GraphicsEnvironment;
+import java.awt.Rectangle;
+import java.awt.Toolkit;
+import java.lang.reflect.Field;
+import java.lang.reflect.Method;
+import java.lang.reflect.Modifier;
+import java.util.Arrays;
+
+/**
+ * Put miscellaneous utilitary static methods here.
```

Comment:
This comment should not document what _we_ should do when we extend this library, but rather what this class does, e.g. "Various utility functions for swing-material."